src/main.rs

```diff
@@ -34,7 +35,9 @@ async fn main() {
 
         for y in 0..GRID_HEIGHT {
             for x in 0..GRID_WIDTH {
+                // this should be a get_alive cell from grid
                 if grid.get(x, y) == conways::CellState::Alive {
+                    // This draws the cell
                     draw_rectangle(x as f32 * 10.0, y as f32 * 10.0, 10.0, 10.0, WHITE);
                 }
             }
```

Comment:
This file is missing a blank line at the end; you should configure your editor to add it automatically.

Reply:
This!
It's important for all files to have blank lines at the end. Some utilities depend on those to tell when a line ends!

Reply:
Will do it in next PR